linera-views/src/dynamo_db.rs

```diff
@@ -191,6 +201,22 @@ fn extract_key_value_owned(
     Ok((key, value))
 }
 
+fn get_bitsize(len: usize) -> usize {
+    let mut power = 128;
+    let mut expo = 1;
+    loop {
+        if len < power {
+            return expo;
+        }
+        power *= 128;
+        expo += 1;
+    }
+}
+
+fn serialize_len(len: usize) -> usize {
```

Comment:
same

Are trying to predict the BCS size? Consider using https://docs.rs/bcs/0.1.5/bcs/fn.serialized_size.html at least for unit tests to verify that your computation is correct.

Reply:
Ok, I use the `serialized_size` here.